src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -2,42 +2,65 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
+using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Azure.ServiceBus;
 using Microsoft.Azure.ServiceBus.Core;
+using Microsoft.Azure.ServiceBus.Management;
 using Microsoft.Azure.WebJobs.Host.Executors;
 using Microsoft.Azure.WebJobs.Host.Listeners;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.WebJobs.ServiceBus.Listeners
 {
-    internal sealed class ServiceBusListener : IListener
+    internal sealed class ServiceBusListener : IListener, IScaleMonitor<ServiceBusTriggerMetrics>
```

Comment:
Since this ServiceBus implementation is more complicated than some of the others, one option to keep the listener code simple would be to move all the monitor stuff into its own ServiceBusScaleMonitor class, and have this listener implement IScaleMonitorProvider and return an instance (as BlobListener does [here](https://github.com/Azure/azure-webjobs-sdk/blob/2c7b2f30b4d197d091f5aa5543ddf65d6a34e689/src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Listeners/BlobListener.cs#L91)).